frontend/src/lib/router.tsx

```diff
@@ -304,13 +306,28 @@ export const ROUTES: {
     sidebar: 'secrets',
     component: () => <SecretDetails />
   },
+  token: {
+    path: '/token',
+    exact: true,
+    name: 'Token',
+    sidebar: null,
+    noAuthRequired: true,
+    component: () => <Auth/>
+  },
+  oidcAuth: {
+    path: '/auth',
+    name: 'OidcAuth',
+    sidebar: null,
+    noAuthRequired: true,
+    component: () => <OIDCAuth/>
+  },
   login: {
     path: '/login',
     exact: true,
     name: 'Login',
     sidebar: null,
     noAuthRequired: true,
-    component: () => <Auth />
+    component: () => <AuthButtons />
```

Comment:
IMO the `AuthButtons` should be called `AuthChooser`, and the `Auth` should be called, `AuthToken`.
We also need to make sure that we skip the `AuthChooser` if only one auth type is available.